CNV-62830: Updated note about IPv6 single-stack support (#94086)
openshift/openshift-docs: virt/vm_networking/virt-networking-overview.adoc

{VirtProductName} provides advanced networking functionality by using custom resources and plugins. Virtual machines (VMs) are integrated with {product-title} networking and its ecosystem.

> You can run {VirtProductName} on a single-stack IPv6 cluster and connect VMs on different nodes through the OVN-Kubernetes localnet or Linux bridge Container Network Interface (CNI) plugin.

Reviewer: This makes it sound like IPv6 is the only supported option, what about

```suggestion
Support of {VirtProductName} on single-stack IPv6 clusters is limited to the OVN-Kubernetes localnet or Linux bridge Container Network Interface (CNI) plugin.
```

Reviewer: Sounds good!
